README.md

```diff
@@ -5,19 +5,118 @@
 [![codecov](https://codecov.io/gh/dojo/cli-test-intern/branch/master/graph/badge.svg)](https://codecov.io/gh/dojo/cli-test-intern)
 [![npm version](https://badge.fury.io/js/%40dojo%2Fcli-test-intern.svg)](https://badge.fury.io/js/%40dojo%2Fcli-test-intern)
 
-The official Dojo 2 test command.
+The official Dojo 2 test command. This package uses Intern to run unit and functional tests against your Dojo project.
 
-**WARNING** This is _beta_ software. While we do not anticipate significant changes to the API at this stage, we may feel the need to do so. This is not yet production ready, so you should use at your own risk.
+## Usage
 
-## Features
+### Prerequisites
 
-Runs tests built using `@dojo/cli-build-webpack`, or `@dojo/cli-build-app`.
+This project is a plugin for the Dojo CLI. Please visit the [Dojo CLI] project for
+ information about the project and how to install.
 
+### Installation
 
-## How do I use this package?
+The use `@dojo/cli-test-intern` in a project, install the packages:
 
-To build and run tests locally, just execute `dojo test` in your project. There are several configuration options, to see what
-options are available run `dojo test -h`
+```bash
+npm install @dojo/cli-test-intern
+```
+
+### Basic Usage
+
+First, build your application and tests using [@dojo/cli-build-app]
+
+```bash
+dojo build app --mode dev
+dojo build app --mode test
+```
+
+Then, run your tests using @dojo/cli-test-intern
+
+```bash
+dojo test -a
+```
+
+There are several configuration options available. To list them run `dojo test -h`.
+
+## Running Tests
+
+Intern supports two types of testing approaches unit and functional. Unit tests are tests run via node and the local
+[Selenium] tunnel and test isolated blocks of code. Functional tests are run using Selenium in the browser and test
+the overall functionality of the software as a user would interact with it.
+
+### Unit tests
+
+Unit tests may be run explicitly with the `-u` flag or as part of a full test run using the `-a` flag
+
+```bash
+dojo test -u
+```
+
+cli-test-intern will execute tests located at `output/tests/unit.js` in node and in Chrome and provide a report 
+listing any failed tests and display a coverage report.
+
+### Functional tests
+
+Functional tests may be run explicitly with the `-f` flag or as part of a full test run using the `-a` flag
+
+```bash
+dojo test -f
+```
+
+cli-test-intern will execute tests located at `./output/test/functional.js` in Chrome using Selenium and provide a
+ report listing any failed tests.
+
+### Testing services
+
+Intern comes with support for running tests remotely on [BrowserStack], [SauceLabs], and [TestingBot]. You may use one
```

Comment:
Are you planning on adding links to BrowserStack, SauceLabs and TestingBot?

Reply:
Ignore me they do link, didn't know that :)

Reply:
They're linked at the bottom.